.github/workflows/build-and-test.yml

```diff
@@ -17,7 +17,7 @@ env:
   MORYX_OPTIMIZE_CODE: "false"
   MORYX_BUILD_CONFIG: "Release"
   MORYX_BUILDNUMBER: ${{github.run_number}}
-  dotnet_sdk_version: '6.x'
+  dotnet_sdk_version: '7.x'
```

Comment:
6 should be fine

Reply:
Is actually necessary, otherwise the wwwroot directory is copied in the staticwebassets as well as content and contentfiles directories within the package 🤔